app/policies/team_policy.rb

```diff
@@ -1,52 +1,41 @@
 class TeamPolicy < BasePolicy
```

Comment:
i didn't touch this file much -- it uses our third, elusive role (as defined by email addresses) so it will be updated the most in the actual roles implementation